homework-submission/homework-week3/react-hyf/react-hyf-client/src/HomeworkWeek3.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import FullName from './components/FullName';
+import OtherInformation from './components/OtherInformation';
+
+export default class HomeworkWeek3 extends Component {
+  render() {
+    return this.props.data.map(elem => (
+      <div className="individual" key={elem.name}>
```

Comment:
An element name may duplicated, Keys in react needs a key to have a unique trigger for the virtual DOM on any change accords, because in some cases it doesn't distinguish between a 100 items as an example :)

so with this comment you may solve the following:
![ezgif com-video-to-gif(1)](https://user-images.githubusercontent.com/25272822/54246797-a75dbe80-4536-11e9-8c68-4d04cb38fceb.gif)

Reply:
Thank you for attracting my attention to this. In fact I have noticed this abnormal behavior especially when I add to-do items with blank descriptions but I didn't know why!